.github/workflows/ci.pre-release-draft-test-sdk-core-subgraph-integration.yml

```diff
@@ -60,4 +60,7 @@ jobs:
     name: Build and test current subgraph release with previous sdk-core versions
```

Comment:
since you are at it, what could be a less hideous file name? Maybe `ci.pre-release-sdk-core` says it all? And it could be a general pattern going forward for any `pre-release-${package}` testing?

Reply:
yeah, it's kinda for both subgraph/sdk-core though, but I can split it into two files and that's a good pattern for naming convention